apps/dapp-console-api/src/abi/Faucet.ts

```diff
@@ -0,0 +1,164 @@
+export const faucetAbi = [
+  {
+    inputs: [{ internalType: 'address', name: '_admin', type: 'address' }],
+    stateMutability: 'nonpayable',
+    type: 'constructor',
+  },
+  {
+    anonymous: false,
+    inputs: [
+      {
+        indexed: true,
+        internalType: 'string',
+        name: 'authModule',
+        type: 'string',
+      },
+      {
+        indexed: true,
+        internalType: 'bytes32',
+        name: 'userId',
+        type: 'bytes32',
+      },
+      {
+        indexed: false,
+        internalType: 'uint256',
+        name: 'amount',
+        type: 'uint256',
+      },
+      {
+        indexed: true,
+        internalType: 'address',
+        name: 'recipient',
+        type: 'address',
+      },
+    ],
+    name: 'Drip',
+    type: 'event',
+  },
+  {
+    inputs: [],
+    name: 'ADMIN',
+    outputs: [{ internalType: 'address', name: '', type: 'address' }],
+    stateMutability: 'view',
+    type: 'function',
+  },
+  {
+    inputs: [
+      {
+        internalType: 'contract IFaucetAuthModule',
+        name: '_module',
+        type: 'address',
+      },
+      {
+        components: [
+          { internalType: 'string', name: 'name', type: 'string' },
+          { internalType: 'bool', name: 'enabled', type: 'bool' },
+          { internalType: 'uint256', name: 'ttl', type: 'uint256' },
+          { internalType: 'uint256', name: 'amount', type: 'uint256' },
+        ],
+        internalType: 'struct Faucet.ModuleConfig',
+        name: '_config',
+        type: 'tuple',
+      },
+    ],
+    name: 'configure',
+    outputs: [],
+    stateMutability: 'nonpayable',
+    type: 'function',
+  },
+  {
+    inputs: [
+      {
+        components: [
+          {
+            internalType: 'address payable',
+            name: 'recipient',
+            type: 'address',
+          },
+          { internalType: 'bytes32', name: 'nonce', type: 'bytes32' },
+        ],
+        internalType: 'struct Faucet.DripParameters',
```

Comment:
I dont think I see the `data` field that was added to the new faucet contract. is this the updated abi or the abi from the previous version of the faucet?